base.rb

```diff
@@ -0,0 +1,17 @@
+def base(base_type,num)
+  sum =  0
+  new_num = num.to_s.reverse.split("")
```

Comment:
I think we can omit `("")`
```suggestion
  new_num = num.to_s.reverse.split
```